Paintroid/build.gradle

```diff
@@ -15,6 +16,11 @@ android {
         testInstrumentationRunner "pl.polidea.instrumentation.PolideaInstrumentationTestRunner"
 		versionCode 1
 		versionName "1.0"
+        testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
+
+        packagingOptions {
+            exclude 'LICENSE.txt'
```

Comment:
what's LICENSE.txt? If not needed for packaging why do we have it in our repository?